iotedgedev/dotnetmoduleprocessor.py

```diff
@@ -8,6 +8,12 @@ def __init__(self, envvars, utility, output, module_dir):
         self.output = output
         self.module_dir = module_dir
         self.exe_dir = self.envvars.DOTNET_EXE_DIR
+       
+        # Fail fast if dotnet is not on path

```

Comment:
Heads up: My dev env is having some trouble, so I wasn't able to test this E2E yet. I was able to test the idea in a reduced, isolated python example.